fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -2903,7 +2903,7 @@ export class FwsCore implements Instrument {
     this.gpwsGsOff.set(SimVar.GetSimVarValue('L:A32NX_GPWS_GS_OFF', 'Bool'));
     this.gpwsSysOff.set(SimVar.GetSimVarValue('L:A32NX_GPWS_SYS_OFF', 'Bool'));
 
-    // fix me use active transponder
+    // fix me use active transponder & check for fault condition when implemented
     const transponder1State = SimVar.GetSimVarValue('TRANSPONDER STATE:1', 'Enum');
     const transponder2State = SimVar.GetSimVarValue('TRANSPONDER STATE:2', 'Enum');
```

Comment:
There's only one transponder on the MSFS side.

Reply:
That's a shame :(